### PR TITLE
feat: add validator for trx addresses

### PR DESCRIPTION
**Pull request by msamsami**
* A new validator is added for the Tron (TRC20) address

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -9,6 +9,22 @@ Note to self: Breaking changes must increment either
...
+_**Features**_
+
+- feat: add validator for trx addresses by @msamsami in [#384](https://github.com/python-validators/validators/pull/384)
+
+_**Maintenance**_
+
+- maint: bump version by @msamsami in [#384](https://github.com/python-validators/validators/pull/384)
+
+**Full Changelog**: [`0.29.0...0.30.0`](https://github.com/python-validators/validators/compare/0.29.0...0.30.0)
+
```
Comment (yozachar) [inline CHANGES.md:27]: Please add a separator after line `27` and after line `43` (like on [line 120](https://github.com/python-validators/validators/pull/384/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R120
)), to group `patch`ed changes together.

```diff
**Full ...

+---

# 0. ...
```

**Review by yozachar — CHANGES_REQUESTED** [review]

**Review by yozachar — APPROVED** [review]
LGTM!

**Comment by yozachar** [comment]
@msamsami, there's is something odd about the `Table of Contents` in the `crypto_address` documentation see: https://yozachar.github.io/pyvalidators/stable/api/crypto_addresses/, any idea why?

**Comment by msamsami** [comment]
> @msamsami, there's is something odd about the `Table of Contents` in the `crypto_address` documentation see: https://yozachar.github.io/pyvalidators/stable/api/crypto_addresses/, any idea why?

The table of contents in the right sidebar? I noticed it just now. I ran the `python package/export TYPE` commands once again. No changes to the files! And it's the same on local—no idea why!

**Pull request merged**